### PR TITLE
Feature/uae 564 79 order comments

### PR DESCRIPTION
**Pull request by ihor-ivchenko**

### CODE REVIEW

File: app/code/ClassyLlama/OrderComments/Plugin/Model/ShippingInformationManagement.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+     *
+     * @param ShippingInformation $subject
+     * @param \Closure $proceed
+     * @param $cartId
+     * @param ShippingInformationInterface $addressInformation
+     *
+     * @return PaymentDetailsInterface $result
+     * @throws InputException
+     */
+    public function aroundSaveAddressInformation(
```
Comment (janepoltoratskaya) [inline app/code/ClassyLlama/OrderComments/Plugin/Model/ShippingInformationManagement.php]: What was the reason to use around? Can it be done with "after" plugin?

File: app/code/ClassyLlama/OrderComments/Observer/QuoteManagement.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * Set order comment value from quote onto order
+     *
+     * @param Observer $observer
+     *
+     * @return void
+     */
+    public function execute(Observer $observer)
+    {
+        $quote = $observer->getEvent()->getQuote();
```
Comment (janepoltoratskaya) [inline app/code/ClassyLlama/OrderComments/Observer/QuoteManagement.php:32]: Please review Magento documentation and follow their approach if it's possible https://devdocs.magento.com/guides/v2.1/ext-best-practices/tutorials/copy-fieldsets.html

File: app/code/ClassyLlama/OrderComments/etc/module.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * Module config
+ *
+ * @category ClassyLlama
+ * @package ClassyLlama_OrderComments
+ * @copyright Copyright (c) 2018 ClassyLlama
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+	<module name="ClassyLlama_OrderComments" setup_version="1.0.0" >
+			<sequence>
```
Comment (janepoltoratskaya) [inline app/code/ClassyLlama/OrderComments/etc/module.xml]: Please review tabs, it looks like there are more than 4 spaces for each.

**Review by janepoltoratskaya — CHANGES_REQUESTED** [review]

File: app/code/ClassyLlama/OrderComments/Observer/QuoteManagement.php (hunk 2 above)
Comment (ihor-ivchenko) [reply app/code/ClassyLlama/OrderComments/Observer/QuoteManagement.php:32]: @janepoltoratskaya I've tried, but this doesn't work for us

**Pull request merged**